docs/reference/specifiers.rst

```diff
@@ -110,12 +111,12 @@ on *region*
 
 **Dependencies**: :prop:`baseOffset` • :prop:`contactTolerance` • :prop:`onDirection`
 
-If :prop:`position` is not already specified with priority 1, positions the *base* of the object uniformly at random in the given `Region`, offset by :prop:`contactTolerance` (to avoid a collision).
+If :prop:`position` is not already specified with priority 1, positions the *base* of the object uniformly at random in the given `Region`, on the :prop:`onSurface` of the given `Object`, or with the base of the object at the given vector. The position is always offset by half of :prop:`contactTolerance` (to avoid a collision).
 The base of the object is determined by adding the object's :prop:`baseOffset` to its :prop:`position`.
 
-Note that while :specifier:`on` can be used with `Region`, `Object` and `Vector`, it cannot be used with a distribution containing anything other than `Region`. When used with an object the base of the object being placed is placed on the target object's `onSurface` and when used with a vector the base of the object being placed is set to that position.
+Note that while :specifier:`on` can be used with `Region`, `Object` and `Vector`, it cannot be used with a distribution containing anything other than `Region`. 
```

Comment:
Is this right? I don't see what goes wrong with a distribution over vectors, and indeed that seems like a totally reasonable use case (e.g. `new Object on Uniform(*my_favorite_base_points)`).

Reply:
Good point, I'll change the semantics/description.